src/features/commentsSlice.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-param-reassign */
+import { LoadingStatuses } from '../enums/LoadingStatuses';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+import { Comment } from '../types/Comment';
+import { handleItemsLoading, handleItemsSuccess } from '../helpers/helpers';
+
+export const fetchComments = createAsyncThunk<Comment[], number>(
+  'comments/fetchComment',
+  async (id: number) => {
+    const data = await getPostComments(id);
+
+    return data;
+  },
+);
+
+export const addComment = createAsyncThunk<Comment, Omit<Comment, 'id'>>(
+  'comments/addComment',
+  (commentData: Omit<Comment, 'id'>) => {
+    return createComment(commentData);
+  },
+);
```

Comment:
```suggestion
export const addComment = createAsyncThunk<Comment, Omit<Comment, 'id'>>(
  'comments/addComment',
  createComment,
);
```
You can pass you callback directly if you don't do anything else